eng/Version.Details.xml

```diff
@@ -18,6 +18,10 @@
       <Uri>https://github.com/dotnet/dotnet</Uri>
       <Sha>fad253f51b461736dfd3cd9c15977bb7493becef</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Current.Manifest-10.0.100" Version="10.0.100">
```

Comment:
we need to remember to add this for every non-1xx branch going forward, do we have something to track this? or even better add it to branching automation